api/src/utils/media/csv/validation/csv-header-validator.ts

```diff
@@ -77,23 +90,35 @@ export const hasRequiredHeadersValidator = (requiredHeaders?: string[]): CSVVali
   };
 };
 
+export type FileRecommendedHeaderValidatorConfig = {
+  file_recommended_columns_validator: {
+    name?: string;
+    description?: string;
+    recommended_columns: string[];
+  };
+};
+
 /**
- * For each `recommendedHeaders`, adds an error if the header is not present in the csv.
+ * For a specified set of recommended columns, adds a warning if the column is not present in the csv.
  *
- * @param {string[]} [recommendedHeaders]
+ * @param {FileRecommendedHeaderValidatorConfig} [config]
  * @return {*}  {CSVValidator}
  */
-export const hasRecommendedHeadersValidator = (recommendedHeaders?: string[]): CSVValidator => {
+export const hasRecommendedHeadersValidator = (config?: FileRecommendedHeaderValidatorConfig): CSVValidator => {
   return (csvWorksheet) => {
-    if (!recommendedHeaders?.length) {
+    if (!config) {
+      return csvWorksheet;
+    }
+
+    if (!config.file_recommended_columns_validator.recommended_columns.length) {
       return csvWorksheet;
     }
```

Comment:
same question here, if not relevant in prev comment then obviously not relevant here

Reply:
They can be combined, I had them split out just for readability.  But could go either way.